apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -101,20 +101,20 @@ public final class AppConfiguration extends BaseAppConfiguration implements Seri
 
   /**
    * The name of the environment variable that should be used to provide the {@link
-   * LoadAppOptions#isFilteringNonNullAndNon2022Benes()} value, which is a bit complex; please see
+   * LoadAppOptions#isFilteringNonNullAndNon2023Benes()} value, which is a bit complex; please see
    * its description for details.
    */
-  public static final String ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES =
-      "FILTERING_NON_NULL_AND_NON_2022_BENES";
+  public static final String ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2023_BENES =
+      "FILTERING_NON_NULL_AND_NON_2023_BENES";
 
   /**
-   * The default value to use for the {@link #ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES}
+   * The default value to use for the {@link #ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2023_BENES}
    * configuration environment variable when it is not set.
    *
    * <p>Note: This filtering option (and implementation) is an inelegant workaround, which should be
    * removed as soon as is reasonable.
    */
-  public static final boolean DEFAULT_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES = false;
+  public static final boolean DEFAULT_RIF_FILTERING_NON_NULL_AND_NON_2023_BENES = false;
```

Comment:
This was a little surprising to me, but I suppose it makes sense that we'd side with `false` signaling that... this shouldn't be done forever, I guess?

Either this needs to be set to `true` or we should go ahead with changes to the configuration and the configuration management to handle 2023 skips instead. I wouldn't mind making those changes if you're open to a PR against *this* branch... How would you like to proceed?

Reply:
I've no strong opinion on which way to handle it; on the one hand, this was defaulted to false because this was never intended to last this long, but I suppose now that we're here we can just default to true cause the end isn't yet in sight for being able to remove this.

I think I'll just adjust this to be true, given the reality of where we are now.